### PR TITLE
Implement task fruit shop

### PR DESCRIPTION
**Pull request by ValeraLemberov**

### CODE REVIEW

File: src/main/java/FruitMain.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    public static void main(String[] args) {
+        GetDataFromFile getDataService = new GetDataFromFileImpl();
+        List<String> data = getDataService.getFromStorage(PATH_TO_DATA);
+        ConvertData convertData = new ConvertDataImpl();
+        List<FruitTransaction> dataConvert = convertData.convert(data);
+        StrategyImplementation implementStrategy =
+                new StrategyImplementationImpl(new HandlersStore(addPatternHandler()));
+        implementStrategy.strategy(dataConvert);
+        WokWithStorageImpl storageService = new WokWithStorageImpl();
```
Comment (sltnsrh) [inline src/main/java/FruitMain.java:32]: WokWithStorageImpl  - do refactoring the name. And it`s will be better to create instance with reference to the parent class type.
```suggestion
        WorkWithStorageDB storageService = new WorkWithStorageImpl();
```

**Review by sltnsrh — CHANGES_REQUESTED** [review]

**Review by sltnsrh — APPROVED** [review]

File: src/main/java/dao/WokWithStorageImpl.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package dao;
+
+import db.Storage;
+import java.util.Map;
+
+public class WokWithStorageImpl implements WorkWithStorageDB {
+
```
Comment (KostyaNikiforov) [inline src/main/java/dao/WokWithStorageImpl.java:7]: Empty line

File: src/main/java/dao/WorkWithStorageDB.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+import java.util.Map;
+
+public interface WorkWithStorageDB {
+    void addInStorage(String fruits,Integer quantity);
+
+    Integer getFromStorage(String fruit);
+
+    Map<String,Integer> getAllFromStorage();
+
```
Comment (KostyaNikiforov) [inline src/main/java/dao/WorkWithStorageDB.java:11]: Empty line

File: src/main/java/service/filework/GetDataFromFileImpl.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package service.filework;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class GetDataFromFileImpl implements GetDataFromFile {
+
```
Comment (KostyaNikiforov) [inline src/main/java/service/filework/GetDataFromFileImpl.java:9]: Empty line

**Review by KostyaNikiforov — APPROVED** [review]

**Pull request closed without merging**